Adding version warnings to any versioned doc w/ deprecated Lock/Auth0.js

jeffreylees opened the pull request:
## Passwordless warnings
- https://auth0-docs-content-pr-5709.herokuapp.com/docs/libraries/lock/v9
- https://auth0-docs-content-pr-5709.herokuapp.com/docs/libraries/lock/v10
- https://auth0-docs-content-pr-5709.herokuapp.com/docs/libraries/auth0js/v7
- https://auth0-docs-content-pr-5709.herokuapp.com/docs/libraries/auth0js/v8

## Newly unversioned connections/passwordless docs

- https://auth0-docs-content-pr-5709.herokuapp.com/docs/connections/passwordless/spa-sms
- https://auth0-docs-content-pr-5709.herokuapp.com/docs/connections/passwordless/spa-email-code
- https://auth0-docs-content-pr-5709.herokuapp.com/docs/connections/passwordless/spa-email-link
- https://auth0-docs-content-pr-5709.herokuapp.com/docs/connections/passwordless/regular-web-app-sms
- https://auth0-docs-content-pr-5709.herokuapp.com/docs/connections/passwordless/regular-web-app-email-code
- https://auth0-docs-content-pr-5709.herokuapp.com/docs/connections/passwordless/regular-web-app-email-link

Also removes old includes that covered similar ground.

mpaktiti reviewed (changes requested):
The review app does not work https://auth0-docs-content-pr-5709.herokuapp.com/

jeffreylees commented:
Of course it doesn't :grin:

Will look into it

Commit:
Adding version warnings to any versioned doc w/ deprecated Lock/Auth0.js

mpaktiti commented on articles/connections/passwordless/regular-web-app-email-code/v8/index.md:
```diff
@@ -52,6 +52,8 @@ You can follow any of the [Regular Web App Quickstarts](/quickstart/webapp) to s
 
 ### Use your own UI
 
+<%= include('../../../../_includes/_version_warning_auth0js') %>
```
`version-warning` can be used only under a version dropdown (that is, at the top of the page).
If the warning is in the middle of the doc you have to use a `warning`

mpaktiti replied:
This is not the only place so please go through all the files and double check

mpaktiti reviewed (changes requested):
Jeff, please list also all the review app URLs I should check. I know that it is troublesome but if you don't do it then I have to search and see what has been updated. Thanks.

Commit:
Updating panels in connection/passwordless docs

jeffreylees commented:
@mpaktiti 

I fixed the panels - all version pages have version-warnings in the appropriate places now.

I added the links to the connections/passwordless docs to the description.

Each versioned page underneath of Lock v9 and v10 are also included identically, but I assumed you didn't want another twenty or so links -  I can go through and list them if you need them though.

Commit:
Adding version number

mpaktiti commented on articles/connections/passwordless/regular-web-app-email-link/v7/index.md:
```diff
@@ -3,7 +3,7 @@ title: Using Passwordless Authentication with a magic link via email on Regular
 # Passwordless Authentication with a magic link via e-mail on Regular Web Apps
 
-<%= include('../../_older-version-alert' %>
+<%= include('../../../../_includes/_version_warning_auth0js' %>
```
The warning says that they should migrate to Auth0.js v9 asap, but [the snippet already uses v9](https://auth0-docs-content-pr-5709.herokuapp.com/docs/connections/passwordless/regular-web-app-email-link/v7#use-your-own-ui)

mpaktiti replied:
I also think this is confusing one way or another. What if someone does not use the custom UI option and he sees this panel telling him to upgrade to Auth0.js v9? For mixed docs (those that include both Lock and Auth0.js) the message should be different (or it should be a `warning` under each respective title)

mpaktiti commented on articles/connections/passwordless/regular-web-app-email-code/v7/index.md:
```diff
@@ -3,7 +3,7 @@ title: Using Passwordless Authentication with a one-time code via email on Regul
 # Passwordless Authentication with a one-time code via e-mail on Regular Web Apps
 
-<%= include('../../_older-version-alert' %>
+<%= include('../../../../_includes/_version_warning_auth0js' %>
```
Script uses v9 already https://auth0-docs-content-pr-5709.herokuapp.com/docs/connections/passwordless/regular-web-app-email-code/v7#use-your-own-ui

mpaktiti commented on articles/connections/passwordless/regular-web-app-sms/v7/index.md:
```diff
@@ -3,7 +3,7 @@ title: Using Passwordless Authentication in a Regular Web App with SMS
 # Authenticate users with a one-time code via SMS in a Regular Web App
 
-<%= include('../../_older-version-alert' %>
+<%= include('../../../../_includes/_version_warning_auth0js' %>
```
script already uses v9 https://auth0-docs-content-pr-5709.herokuapp.com/docs/connections/passwordless/regular-web-app-sms/v7#use-your-own-ui (plus there is a Lock section before that)

mpaktiti reviewed (changes requested):
Hey, I left some comments. I didn't check all of the docs, so please make sure that any adjustments are applied everywhere.

Commit:
Removing versioned documents for connections/passwordless

Also modifying some of the content in these pages

mpaktiti committed:
Merge branch 'master' into version-warning-panels

mpaktiti commented on articles/connections/passwordless/regular-web-app-email-code.md:
```diff
@@ -3,19 +3,19 @@ title: Using Passwordless Authentication with a one-time code via email on Regul
 ## Setup
 
-<%= include('../../_setup-email') %>
+<%= include('_setup-email') %>
```
this is H4, while the `Setup` is H2

mpaktiti commented on articles/connections/passwordless/regular-web-app-email-code.md:
```diff
@@ -3,19 +3,19 @@ title: Using Passwordless Authentication with a one-time code via email on Regul
+<%= include('_setup-email') %>
 
-<%= include('../../_setup-callback', {spa:false} )%>
+<%= include('_setup-callback', {spa:false} )%>
```
same, goes from H2 to H4

mpaktiti commented on articles/connections/passwordless/regular-web-app-email-link.md:
```diff
@@ -3,19 +3,19 @@ title: Using Passwordless Authentication with a magic link via email on Regular
 ## Setup
 
-<%= include('../../_setup-email') %>
+<%= include('_setup-email') %>
```
goes from H2 to H4

mpaktiti commented on articles/connections/passwordless/regular-web-app-email-link.md:
```diff
@@ -3,19 +3,19 @@ title: Using Passwordless Authentication with a magic link via email on Regular
+<%= include('_setup-email') %>
 
-<%= include('../../_setup-callback', {spa:false} ) %>
+<%= include('_setup-callback', {spa:false} ) %>
```
goes from H2 to H4

mpaktiti commented on articles/connections/passwordless/regular-web-app-sms.md:
```diff
@@ -3,19 +3,19 @@ title: Using Passwordless Authentication in a Regular Web App with SMS
 ## Setup
 
-<%= include('../../_setup-sms-twilio') %>
+<%= include('_setup-sms-twilio') %>
```
gos from H2 to H4

mpaktiti commented on articles/connections/passwordless/regular-web-app-sms.md:
```diff
@@ -3,19 +3,19 @@ title: Using Passwordless Authentication in a Regular Web App with SMS
+<%= include('_setup-sms-twilio') %>
 
-<%= include('../../_setup-callback', {spa:false} ) %>
+<%= include('_setup-callback', {spa:false} ) %>
```
gos from H2 to H4

mpaktiti commented on articles/connections/passwordless/spa-email-code.md:
```diff
@@ -4,19 +4,19 @@ description: Learn how to authenticate users with a one-time-code using email in
 ## Setup
 
-<%= include('../../_setup-email') %>
+<%= include('_setup-email') %>
```
gos from H2 to H4

mpaktiti commented on articles/connections/passwordless/spa-email-code.md:
```diff
@@ -4,19 +4,19 @@ description: Learn how to authenticate users with a one-time-code using email in
+<%= include('_setup-email') %>
 
-<%= include('../../_setup-cors') %>
+<%= include('_setup-cors') %>
```
gos from H2 to H4

mpaktiti commented on articles/connections/passwordless/spa-email-link.md:
```diff
@@ -3,19 +3,19 @@ title: Using Passwordless Authentication with a Magic Link via email on SPA
 ## Setup
 
-<%= include('../../_setup-email') %>
+<%= include('_setup-email') %>
```
goes from H2 to H4

mpaktiti commented on articles/connections/passwordless/spa-email-link.md:
```diff
@@ -3,19 +3,19 @@ title: Using Passwordless Authentication with a Magic Link via email on SPA
+<%= include('_setup-email') %>
 
-<%= include('../../_setup-callback', {spa:true} ) %>
+<%= include('_setup-callback', {spa:true} ) %>
```
goes from H2 to H4

mpaktiti commented on articles/connections/passwordless/spa-sms.md:
```diff
@@ -4,19 +4,19 @@ description: Learn how to authenticate users with a one-time-code using SMS in a
 ## Setup
 
-<%= include('../../_setup-sms-twilio') %>
+<%= include('_setup-sms-twilio') %>
```
goes from H2 to H4

mpaktiti commented on articles/connections/passwordless/spa-sms.md:
```diff
@@ -4,19 +4,19 @@ description: Learn how to authenticate users with a one-time-code using SMS in a
+<%= include('_setup-sms-twilio') %>
 
-<%= include('../../_setup-cors') %>
+<%= include('_setup-cors') %>
```
goes from H2 to H4

mpaktiti reviewed (changes requested):
Headers level mess up. Also, since we removed so many docs, can you please add an update feed entry? No need to mention the warnings you added on the other docs, but you should mention the versions that were removed.

Commit:
Fixing heading levels and adding an updates feed entry

Commit:
Merge remote-tracking branch 'origin/version-warning-panels' into version-warning-panels
